physionet-django/physionet/settings/base.py

```diff
@@ -534,6 +534,7 @@ class StorageTypes:
 
 SOURCE_CODE_REPOSITORY_LINK = config('SOURCE_CODE_REPOSITORY_LINK',
                                      default='https://github.com/MIT-LCP/physionet-build')
+MAX_TRAINING_REPORT_UPLOAD_SIZE_MB = config('MAX_TRAINING_REPORT_UPLOAD_SIZE_MB', cast=int, default=1)
```

Comment:
HI, This seems some kind of mistake, i remember removing the MB stuff and doing bytes as Benjamin suggested. 

it might happened because i was poking around multiple branches. Pushing the changes with the fix asap

Reply:
But yes, we want everything in KB as per discussion on the previous closed PR